modules/develop/pages/config-topics.adoc

```diff
@@ -82,18 +82,44 @@ The cleanup policy determines how to clean up the partition log files when they
 * `compact` compacts the data by only keeping the latest values for each KEY. For details on compaction in Redpanda, see xref:manage:cluster-maintenance/compaction-settings.adoc[Compaction settings].
 * `compact,delete` combines both methods.
 
-Unlike compacted topics which keep only the most recent message for a given key, topics configured with a delete cleanup policy provide a running history of all changes for those topics. 
+Unlike compacted topics, which keep only the most recent message for a given key, topics configured with a `delete` cleanup policy provide a running history of all changes for those topics. 
 
-Suppose you configure a topic using the default cleanup policy `delete`, and you want to change the policy to `compact`. Run the `rpk topic alter-config` command as shown:
+For example, to change a topic's policy to `compact`, run:
 
 [,bash]
 ----
 rpk topic alter-config [TOPICS…] —-set cleanup.policy=compact
 ----
 
+=== Configure write caching
+
+Write caching is a relaxed mode of xref:develop:produce-data/configure-producers.adoc#acksall[`acks=all`] that provides better performance at the expense of durability. It acknowledges a message as soon as it is received and acknowledged on a majority of brokers, without waiting for it to fsync to disk. This provides lower latency while still ensuring that a majority of brokers acknowledge the write. 
+
+Write caching applies to user topics. It does not work with transactions or consumer offsets:  transactions and offset commits are always fsynced. 
+
+NOTE: For clusters in xref:reference:rpk/rpk-redpanda/rpk-redpanda-mode.adoc#development-mode[development mode], write caching is enabled by default. For clusters in production mode, it is disabled by default. 
+
+Only enable write caching on workloads that can tolerate some data loss in the case of multiple, simultaneous broker failures. Leaving write caching disabled safeguards your data against complete data center or availability zone failures. 
+
+==== Configure at cluster level
+
+To enable write caching in production mode, set the cluster-level property xref:reference:cluster-properties.adoc#write_caching[`write_caching`]:
```

Comment:
This sentence has a bit of ambiguity to whether this is required to be able to set topic level configuration or whether the cluster property changes the topic default.

I'd consider rewording a bit.

At least to something like "To enable write caching by default for all user topics ... set the cluster-level property"